tests/integration/vxlan/01-vxlan-bridging.yml

```diff
@@ -11,48 +11,77 @@ message: |
   Please note it might take a while for the lab to work due to
   STP learning phase
 
+defaults.sources.extra: [ ../wait_times.yml ]
+
 groups:
   _auto_create: True
   hosts:
-    members: [ h1, h2, h3, h4 ]
+    members: [ h1, h2, h3, h4, h1a, h3a ]
     device: linux
     provider: clab
   switches:
-    members: [ s1, s2 ]
+    members: [ dut, s2 ]
     module: [ vlan, vxlan, ospf ]
+  probes:
+    members: [ s2 ]
+    device: frr
+    provider: clab
 
 vlans:
   red:
+    vni: 5000
```

Comment:
Do we have/need a device quirk that warns about this?

Reply:
We don't have one, but we could definitely use it. Thank you!

Reply:
Implemented in #2283